server/openejb-cxf-rs/src/main/java/org/apache/openejb/server/cxf/rs/event/ExtensionProviderRegistration.java

```diff
@@ -25,10 +26,12 @@
 // this event can allow to add/remove/resort providers
 @Event
 public class ExtensionProviderRegistration {
+    private final Server server;
     private final List<Object> providers;
     private final AppContext appContext;
 
-    public ExtensionProviderRegistration(final AppContext ctx, final List<Object> existings) {
+    public ExtensionProviderRegistration(final Server server, final AppContext ctx, final List<Object> existings) {
```

Comment:
Yes we can rename the parameter and I need to revert the change and not pass the Server because when the event is created the server does not exist yet.